### ISSUE TITLE
crash when attempting to grabe length of address names

### ISSUE THREAD

**Issue by equilet** [CNMAT/CNMAT-odot#375]
...seems like this is not a copy but maybe a reference?  Not sure actually....

<pre><code>
----------begin_max5_patcher----------
720.3ocyWt0aaBCEG+YxmBK+TqDMEatDXR6087deqJx.tozA1LroJcU8697E
HWFnDRaZVkhrvGNGe7+i+cbRdYlCLkulJffu.9Avw4kYNNFSZCNcycfUj0Yk
DgwMXFuphxjPW66jz0Ri8TNup2Hqsh2JKoRSHdcVKxMNxSe7Fjeuq0DY1CEr
UKanYR69HJNbtmKHvSOtHQOhUOCta6pWv5Wbj11qyloGbemRfvxAD.xE656N
e9lcXCUn7lHK3rkkELZFukYB.OY4hNfbC7+uJW7zOz7NfJvQ3spH3xenMYUf
OfHrU+ONMvmqTQMWPGnhaI5kz8mraSMfk5gL0CaZS1i67GUqnQNw1H164Lon
3OTii5SJqYa3xmqo1B.DtQriTdBS1o9r.oGCQ6TezI4dRFc2J+d0r9FFA4IZ
9xzVVdIcYIksR9f90QAi89bhjXyuenKHI1EfPKzCpzi7LCZadHWURO9GDVo.
U7IK5sDDnCObnmiEU7whBOVTIGKJevcuIdhtttQAU4TsiCgp77mEfuBVQkpG
U2iInhqt1EnnKhxLQP+8UpjaOAtpy8a.nqudTta766vC4tjSE69VaY42IY+h
p5u2a1DvQjYgA3.CPFDcJ.4aoINsUJ4L3T6AWr89lFREURaVRYjzx81TCJHo
D1pCJ9.uctuEGzOdNuq5Hr0N.DxyBUoiAUuWXJ9h.Sc0yNXBE6+oDlh9XfIL
N4SFLYPosHz6ifBuHDTWQrmfB8tLDzIVZgSsnEbYtCGMrl4i+j004+wz0Y09
o2zY7vzJ7O+wMSNz12W5BdaSV+tp6qj.aySNUHKXl+dyN9Da8Yzh6TySzDxS
3YHO9SHOAmg7n+4zGMQICRj87hTW+DsQz4rIGJn7QdidZrqYZAyN0vTvF5SE
89aZPgjFECIU.TaiYeAWGYaSgUp1+FVaQGqnTmJkFfmoHVQcWGjouX1qy9ax
YXKM
-----------end_max5_patcher-----------
</code></pre>

**Comment by equilet** [comment]
```
{
	"version" : "Version 8.0.2 (a3679a2) (64-bit mac)",
	"platform" : "mac",
	"arch" : "x64",
	"osversion" : "Mac OS X Version 10.14.1 (Build 18B75) x86_64",
	"samplerate" : 44100,
	"iovs" : 128,
	"sigvs" : 64,
	"scheduler_in_audio_interrupt" : "off",
	"audio_drivername" : "Core Audio",
	"audio_driver_subname" : "",
	"license" : "permanent full",
	"machine_id" : "06e024609662aa9dc4f572b2cdddd559",
	"eventinterval" : 2,
	"schedinterval" : 1.0,
	"overdrive" : "on",
	"pollthrottle" : 40,
	"queuethrottle" : 100,
	"sysqelemthrottle" : 1000,
	"refreshrate" : 33.333332061767578,
	"schedslop" : 25.0,
	"eventprobing" : 0,
	"mixerparallel" : "off",
	"mixercrossfade" : 0,
	"mixerlatency" : 30.0,
	"mixerramptime" : 10.0,
	"videoengine" : "avf",
	"glengine" : "gl2",
}
```

**Comment by maccallum** [comment]
Yeah, no crash for me either on 8.0.1. Same behavior as Rama—it prints an error about subtracting a number from a string.
